wandb/sdk/artifacts/artifact.py

```diff
@@ -884,36 +896,57 @@ def _update(self) -> None:
                 for alias in self._aliases
             ]
 
-        mutation = gql(
-            """
-            mutation updateArtifact(
-                $artifactID: ID!,
-                $description: String,
-                $metadata: JSONString,
-                $aliases: [ArtifactAliasInput!]
+        mutation_template = """
+        mutation updateArtifact(
+            $artifactID: ID!,
+            $description: String,
+            $metadata: JSONString,
+            _TTL_DURATION_SECONDS_TYPE_
+            $aliases: [ArtifactAliasInput!]
+        ) {
+            updateArtifact(
+                input: {
+                    artifactID: $artifactID,
+                    description: $description,
+                    metadata: $metadata,
+                    _TTL_DURATION_SECONDS_VALUE_
+                    aliases: $aliases
+                }
             ) {
-                updateArtifact(
-                    input: {
-                        artifactID: $artifactID,
-                        description: $description,
-                        metadata: $metadata,
-                        aliases: $aliases
-                    }
-                ) {
-                    artifact {
-                        id
-                    }
+                artifact {
+                    id
                 }
             }
-            """
-        )
+        }
```

Comment:
Nit: increase the indentation by 2 spaces so that it easy to spot where the string begins and ends.